DTS-STN/Service-Canada-Labs
# ICS Overview and Articles Pages

# [Integrated Channel Strategy - New](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities)

This PR includes the creation of the ICS overview and article pages. Currently there are no article pages in AEM but the setup is there for when they are added.

## Test Instructions

1. Navigate to the "Making it easier to get benefits" page
2. See that it renders without errors and is bilingual
